apps/emqx/src/emqx_schema.erl

```diff
@@ -2066,6 +2066,10 @@ desc("ocsp") ->
 desc(_) ->
     undefined.
 
+%% TODO: remove after e5.0.2
+hidden("ocsp") -> true;
```

Comment:
With this meta and [this patch](https://github.com/emqx/hocon/pull/237), the `ocsp` struct is removed from the generated JSON.